three.js/Makefile

```diff
@@ -17,7 +17,12 @@ minify-main: build-main
 
 build: build-main
 
-minify: minify-main
+minify-lean: build-lean
```

Comment:
i think we should delete the lean part again, if i'm not wrong we deleted because not necessary anymore.

Reply:
ok sorry I re-added for mistake. PLease delete

Reply:
ok i will delete this.